### ISSUE TITLE
Life-Mending appears to not be working

### ISSUE THREAD

**Issue by Steampunkgears** [Shadows-of-Fire/Apotheosis#742]
Hi, I seem to run into a issue where life mending does not work. I applied it to an iron pick fyi. I added other enchants like unbreaking and Fervor and those work fine. My game version is 1.18.2
I know your mod has nothing to do with vanilla mending but that appears broken to. I dont think I have another mod that would conflict with this but just in case here is the mod list my server uses. Please let me know if you see a mod that has known compatibility issues with yours. Thanks for your time.
[https://www.curseforge.com/minecraft/modpacks/ninth-exiles-modpack/files/4153805](url)

**Comment by Shadows-of-Fire** [comment]
Life mending consumes incoming healing to restore durability.  Is that behavior not happening?  It does not occur when you are at full health.

**Comment by Steampunkgears** [comment]
Let me try again, I tried fall damage and then taking a healing potion

**Comment by Shadows-of-Fire** [comment]
The item also has to either be in-hand or equipped as armor (same as mending)

**Comment by Steampunkgears** [comment]
OK as soon as you said it had to be equipped I facepalmed. Thanks for that. However Im still perplexed as regular mending wont even apply through the anvil interface. Do you have any clue what might be going on here? 
here is a screenshot of the tool tip ui of both before applied [https://imgur.com/a/jJZ1ik7](url)
The photo of the iron pick UI is literally the photo of it being overed over while everything is plugged into the anvil and even after I have tested XP gains have no effect on the pick.

**Comment by Shadows-of-Fire** [comment]
No clue, apoth doesn't modify vanilla mending by any means, so you might need to check with other mods

**Comment by Steampunkgears** [comment]
Thank you for your time...At least your mending works.
